src/lua/bcc/vendor/helpers.lua

```diff
@@ -1,3 +1,29 @@
+do
+  local ffi = require("ffi")
+  local ptrtype = ffi.typeof("uint64_t")
+  local strformat = string.format
+  function string.format(format, ...)
```

Comment:
This is a bit of Rocky's Horror Show, but IMO it's worth it to let probe authors use `%p` like they'd expect to.